platzky/blog/blog.py

```
@@ -61,14 +71,13 @@ def get_post(post_slug):
                 comment_sent=request.args.get("comment_sent"),
             )
         except ValueError:
-            return page_not_found(f"no post with slug {post_slug}")
-        except Exception as e:
-            return page_not_found(str(e))
+            abort(404)
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)